manual/io/index.md

```diff
@@ -1,41 +1,24 @@
 ---
-title: I/O
+title: I/O Concepts
 layout: single
 sidebar:
   nav: "manual"
 toc: true
 toc_sticky: true
 ---
 
-This page explains what needs to be done to store your classes using ROOT, be it object-wise or as columnar data, see → [trees]({{ '/manual/trees/' | relative_url }}).
-
-It also explains the mechanisms and concepts behind ROOT's I/O facilities, i.e. how ROOT converts your objects into a stream of bytes and back.
-
+This page explains the mechanisms and concepts behind ROOT's I/O facilities, i.e. how ROOT converts your objects into a stream of bytes and back.
+It assumes that you have read the introduction on → [ROOT files]({{ '/manual/root_files/' | relative_url }}), → [I/O of custom classes]({{ '/manual/io_custom_classes/' | relative_url }}), and → [Trees]({{ '/manual/trees/' | relative_url }}).
 
 ## ROOT files, directories, and keys
 
 ROOT stores data in ROOT files ({% include ref class="TFile" %}), see → [ROOT files]({{ '/manual/root_files/' | relative_url }}).
-It does so in machine independent format (IEEE floating point, Big Endian byte ordering).
+It does so in a machine independent format.
 
 Similar to a file system directory, a `TFile` can contain directories ({% include ref class="TDirectory" %}) and objects, accessible through the directory's _keys_ ({% include ref class="TKey" %})).
 A `TFile` is a directory itself: it inherits from `TDirectory`.
 
-Directories can be "entered" by [`TDirectory::cd()`](https://root.cern/doc/master/classTDirectory.html#af42d9b50db9b4575b13ea265ad8995fb); created by [`TDirectory::mkdir()`](https://root.cern/doc/master/classTDirectory.html#aad2cf362d9574db6538804afa9794827); removed by [`TDirectory::rmdir()`](https://root.cern/doc/master/classTDirectory.html#a528126c0a528d7f7a5888c7ee74ae41d).
-
-### Opening a ROOT file
-
-Use [TFile::Open()](https://root.cern/doc/master/classTFile.html#ad8870806a04da2c2f4aa02bee4ec6833){:target="_blank"} to open a ROOT file.
-While this operation might return a valid pointer to a `TFile` object,
-this object might not be able to access data, for instance because ROOT was unable to open the file in the filesystem.
-Use [TObject::IsZombie()](https://root.cern/doc/master/classTObject.html#aaa8418b9b6692a12d8d0e500c51911bf){:target="_blank"} to check whether the ROOT file was successfully opened.
-
-{% highlight C++ %}
-std::unique_ptr<TFile> file( TFile::Open("file.root") );
-if (file->IsZombie()) {
-   std::cerr << "Error opening file" << endl;
-   exit(-1);
-}
-{% endhighlight %}
+Directories can be "entered" by [`TDirectory::cd()`](https://root.cern/doc/master/classTDirectory.html#af42d9b50db9b4575b13ea265ad8995fb); created by [`TDirectory::mkdir()`](https://root.cern/doc/master/classTDirectory.html#aad2cf362d9574db6538804afa9794827); and removed by [`TDirectory::rmdir()`](https://root.cern/doc/master/classTDirectory.html#a528126c0a528d7f7a5888c7ee74ae41d).
```

Comment:
What is the process to maintain the commit information that is embedded here? (I.e. if TDirectory.h changes, how do we remember to update this commit number)?

Reply:
the CI at `root-project/web` complains next time anyone tries to modified the website